talentprotocol/web-dapp
# feat: add opportunities follow up emails

## Summary

This PR adds four types of "opportunity follow up" emails when career needs change.

## Notion link

https://talentprotocol.notion.site/Add-opportunities-follow-up-email-b833ae38d1f8447ba75e68fdd1e3d465

## How to test?

- Add "Finding and hiring talent" as a career need
- Should receive the below **Looking to hire talent?** email
- Remove "Finding and hiring talent" as a career need
- Should receive the below **Did you find talent?** email
- If you repeat the above steps, you shouldn't receive either email again (until three months later)

![Screenshot 2022-11-19 at 20 18 23](https://user-images.githubusercontent.com/33064059/202869928-16fd2618-0a10-4f77-85e4-d2ebe2f49114.png)

![Screenshot 2022-11-19 at 20 13 29](https://user-images.githubusercontent.com/33064059/202869753-8dd9c750-9ae8-4e19-9598-c1c2c9670acd.png)

---

- Add "Freelancing or contract roles", "Full-time Roles", or "Part-time Roles" as a career need
- Should receive the below **We have open roles for you!** email
- Remove all of the above as a career need
- Should receive the below **Did you just land a new role?** email
- If you repeat the above steps, you shouldn't receive either email again (until three months later)

![Screenshot 2022-11-19 at 20 21 18](https://user-images.githubusercontent.com/33064059/202870031-f9c2c131-a8f7-4048-84af-93f8a7f0889c.png)

![Screenshot 2022-11-19 at 20 17 21](https://user-images.githubusercontent.com/33064059/202869889-b5d96a57-79e7-4cb5-aa0b-9c2252393b0b.png)
